### DISCUSSION TITLE
navidrom aplication

### DISCUSSION THREAD

**Discussion by Josuate** [navidrome/navidrome#3170] [General]
good day, I would like to consult can you recommend me an open source application for navidrome on android and ios.

**Comment by deluan** [comment]
Symfonium is really good, but not Open Source. Take a look at this list, most of the options are Open Source: https://www.navidrome.org/docs/overview/#apps

**Comment by JPGuillemin** [comment]
Airdrome !! https://github.com/JPGuillemin/Airdrome
